root.spec

```diff
@@ -146,7 +163,43 @@ done
 export ROOT_INCLUDE_PATH
 
 export ROOTSYS=%i
+
 make INSTALL="$cp" INSTALLDATA="$cp" install
+# Handle debug symbols. We always have a debug subpackage for debug symbols so
+# no point in complicating the standard builds with this.
+
+%if "%{?subpackageDebug:set}" == "set"
+  ELF_DIRS="%i/lib %i/bin"
+  DROP_SYMBOLS_DIRS=""
+
+# optimise the debug symbols, compress them, and split them into separate file
```

Comment:
I assumed that the scram build produced common-symbols.debug. The root build did not produce a file name common-symbols.debug so I removed this. If you know how to do the same thing for root please do so.